### PR TITLE
CUMULUS-292 Create Contributing.md

### PR DESCRIPTION
**Pull request by laurenfrederick**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+If you have any questions or ideas, or notice any problems or bugs, first [search open issues](https://github.com/cumulus-nasa/cumulus-message-adapter/issues) to see if the issue has already been submitted. We may already be working on the issue. If you think your issue is new, you're welcome to [create a new issue](https://github.com/cumulus-nasa/cumulus-message-adapter/issues/new).
+
+## Pull Requests
+
+If you want to submit your own contributions, follow these steps:
+
+* Fork the Cumulus Message Adapter repo
+* Create a new branch from the branch you'd like to contribute to
+* If an issue does't already exist, submit one (see above)
+* [Create a pull request](https://help.github.com/articles/creating-a-pull-request/) from your fork into the target branch of the cumulus-nasa/cumulus repo
```
Comment (abarciauskas-bgse) [inline CONTRIBUTING.md]: This should be `of the cumulus-nasa/cumulus-message-adapter` repo?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+* Upon submission of a pull request, the Cumulus development team will review the code
+* The request will then either be merged, declined, or an adjustment to the code will be requested
+
+## Guidelines
+
+We ask that you follow these guidelines with your contributions:
+
+### Tests
+
+All of the automated tests for this project need to pass before your submission will be accepted. See the README for instructions on how to run tests and verify that the tests pass. If you add new functionality, please consider adding tests for that functionality as well.
```
Comment (abarciauskas-bgse) [inline CONTRIBUTING.md]: I would remove "consider" ;)

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+### Commits
+
+* Make small commits that show the individual changes you are making
+* Write descriptive commit messages that explain your changes
+
+Example of a good commit message;
```
Comment (abarciauskas-bgse) [inline CONTRIBUTING.md]: Should this be a colon?
```
Example of a good commit message:
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (laurenfrederick) [reply CONTRIBUTING.md]: Ahhhh I didn't change that one on any of them!! Good catch, thank you!

File: CONTRIBUTING.md (hunk 2 above)
Comment (laurenfrederick) [reply CONTRIBUTING.md]: Not my wording, probably Patrick's, so I am going to leave it.

File: CONTRIBUTING.md (hunk 3 above)
Comment (laurenfrederick) [reply CONTRIBUTING.md]: Yep thank you! Fixed.

**Pull request merged**